wazo_websocketd/main.py

```diff
@@ -21,18 +21,19 @@ def main():
         config['log_file'], debug=config['debug'], log_level=config['log_level']
     )
     xivo_logging.silence_loggers(['urllib3'], logging.WARNING)
+    xivo_logging.silence_loggers(['aioamqp'], logging.CRITICAL)
```

Comment:
Does using WARNING instead of CRITICAL continue to polluting logs?
I always feel uncomfortable to remove WARNING log without good reason (and it's not mentioned in the commit)